chromium/external/README.md

```diff
@@ -0,0 +1,42 @@
+# External dependencies
+
+This directory contains files that are external dependencies for HTTPS Everywhere, with instructions on how to get or remake them.
+
+## CodeMirror 5.31.0
+
+```
+$ npm install uglify-js@3.1.9
+$ npm install uglifycss@0.0.27
+$ curl -o codemirror.zip https://codemirror.net/codemirror-5.31.0.zip
+$ unzip codemirror.zip
+```
+
+### codemirror-5.31.0.min.js
+
+```bash
+uglifyjs \
+  --compress \
+  --mangle \
+  --comments '/copyright|license/' \
+  --output codemirror-5.31.0.min.js \
```

Comment:
Relating to the suggestion above of adding each external dependency to its own unique path, we'll want to change these output paths to `external/codemirror/`...

Reply:
See https://github.com/EFForg/https-everywhere/pull/13560#discussion_r151280705.